org.eclipse.ui.intro/src/org/eclipse/ui/internal/intro/impl/swt/PageStyleManager.java

```diff
@@ -377,7 +377,7 @@ public boolean getShowLinkDescription() {
 		}
 		if (value == null)
 			value = "true"; //$NON-NLS-1$
-		return value.toLowerCase().equals("true"); //$NON-NLS-1$
+		return value.equalsIgnoreCase("true"); //$NON-NLS-1$
```

Comment:
What about `Boolean.parseBoolean()`? This would be even more expression and would save the extra string and this the NLS suppression.
You could also incorporate the null case the make it to something like
`return value == null || Boolean.parseBoolean(value)`

Reply:
ok, thanks.